pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple image files ie. resizing, cropping, flipping, adding effects.
```

Comment:
Perhaps mention that it alters the original image (i.e. as compared with `convert`, which saves a new file)?